Example/SwiftPrettyPrintExample/Source/AppDelegate.swift

```diff
@@ -6,11 +6,17 @@
 //  Copyright © 2020 Yusuke Hosonuma. All rights reserved.
 //
 
+import SwiftPrettyPrint
 import UIKit
 
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
     func application(_: UIApplication, didFinishLaunchingWithOptions _: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        // -------------------------
+        // Configure format-options
+        // -------------------------
+        Debug.sharedOption = Option(indent: 4)
```

Comment:
This comment is nothing to do with this PR..
`Option` is a general word. 
so It is better that this is in `Debug`'s name space.

https://github.com/YusukeHosonuma/SwiftPrettyPrint/blob/887741e497ad1d6bc520a934cb07fb92ea82963b/Sources/API/Option.swift

Reply:
I have got the same impression.
`Option` could be in some module explicitly.

Reply:
Thanks for feedbacks!!
I also thought to, I'll fix it.

Reply:
Fixed!! 108ea4a